### DISCUSSION TITLE
Request for VDA 5050 Submodule Upgrade to Support OpenTCS 6.0.0

### DISCUSSION THREAD

**Discussion by BenYaho** [openTCS/opentcs#158] [Q&A] [answered]
Hello OpenTCS Team,

I am currently using OpenTCS version 5.17.1 along with the VDA 5050 submodule version 0.0.20. With the recent release of OpenTCS 6.0.0, I was wondering if there are any plans to update the VDA 5050 submodule to support this new version. If an upgrade is in the works, I would greatly appreciate it if you could share any details on the expected timeline.

Thank you so much for your ongoing efforts and support. I’m looking forward to hearing from you.

**Comment by swltr** [comment, marked as answer]
@BenYaho The upgrade to openTCS 6 is currently being worked on. A VDA5050 driver release including that upgrade should arrive within the next few days.

**Reply by swltr** [reply]
@BenYaho Note that there is a new VDA5050 driver release, 0.21. It is based on openTCS 6.0.
